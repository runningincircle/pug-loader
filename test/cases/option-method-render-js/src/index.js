// require with alias 'Views'
const html = require('Views/widget.pug?{"a":10,"b":"abc"}');

console.log(html);