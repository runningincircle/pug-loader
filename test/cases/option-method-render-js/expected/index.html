<h1>Hello world!</h1><span>Param[a]: 10</span><span>Param[b]: abc</span>